src/spatialmath/utils.rs

```diff
@@ -0,0 +1,62 @@
+use nalgebra::Quaternion;
+
+pub fn to_euler_angles(quat: Quaternion<f64>) -> [f64;3] {
```

Comment:
I don't love returning an array here as opposed to creating a `euler_angles` struct with `roll`, `pitch`, and `yaw` as explicitly named fields. Can we fix in this pr? I understand if this is a hassle to do immediately but I think long term we should really be more precise with our type signatures, so at a minimum we should create a new ticket and add a TODO comment.

Reply:
I can do it in this PR, I assume you want me to do this with axis angles as well then?

Reply:
D'oh, missed the `axis_angles` case, yes doing it there too would be great thank you!